HISTORY.md

```diff
@@ -1,3 +1,71 @@
+## [15.0.2](https://github.com/theforeman/puppet-foreman/tree/15.0.2) (2020-08-03)
```

Comment:
If you modify the latest version (15.1.0 now) I'd also add it to HISTORY.md immediately. You'll need to do that on whenever you do the next version and you can be kind to your future self.